services.go

```diff
@@ -258,3 +245,23 @@ func WithInMemoryServices(ic *plugin.InitContext) ClientOpt {
 		return nil
 	}
 }
+
+func WithSandboxers(ic *plugin.InitContext) ClientOpt {
```

Comment:
Maybe the name `WithSandboxControllers` makes more sense.

Reply:
changed to `WithInMemorySandboxControllers`